src/host/main.cpp

```diff
@@ -112,6 +112,10 @@ int main(int argc, char** argv)
       "Logging level for the enclave code")
     ->transform(CLI::CheckedTransformer(log_level_options, CLI::ignore_case));
 
+  std::string enclave_file_path;
+  app.add_option(
+    "--enclave-file", enclave_file_path, "Path to enclave application");
```

Comment:
I think it would be nice to indicate why this is set as a CLI argument rather than in the config here. Maybe a `"(security critical)"` label here, and then a quick sentence in the output of `--help`?